views/show.tmpl

```diff
@@ -25,19 +25,19 @@
   <div class="container p-3">
     <div class="row">
       <div class="col-12 col-sm-5 col-lg-3">
-         <img class="img-fluid show-img" src="
-         {{if .Show.Photo}}
-         https://ury.org.uk{{.Show.Photo}}
-         {{else}}
-         /images/show-current.png
-         {{end}}"
-         alt="{{.Show.Title}} Logo">
+        <img class="img-fluid show-img" src="
+        {{if .Show.Photo}}
+        https://ury.org.uk{{.Show.Photo}}
```

Comment:
pretty sure this shouldn't be hardcoded

Reply:
Hmm, the copy paste is probably left over from before the {{url blah}} function.

Reply:
@LordAro ah, I think looking back, it's done like this because of the overall assumed role that if 2016-site doesn't provide the file/page itself, it should be hardcoded.

Using a full URL var would be better in the future, but we haven't worked out yet how to easily integrate stuff from PageContext without bombing out of PageData every time.